docker/Makefile

```diff
@@ -168,6 +168,36 @@ ifeq ($(shell echo $(PUSH) | tr [:upper:] [:lower:] ),true)
 	docker push harbor.h2o.ai/opsh2oai/h2o-3/dev-python-3.6-jdk-$*$(GPU_SUFFIX):$(VERSION)
 endif
 
+ifneq ($(CI), 1)
+dev-python-3.7-openjdk-7: dev-python-3.7
+endif
+dev-python-3.7-openjdk-7:
```

Comment:
is it for the "perverts" who want to use bleeding-edge Python with 8yo Java? :)
Meaning, is it worth providing/supporting this?

Reply:
@michalkurka will have to tell if it's required, but there was an issue (I'm not sure now what was the issue though) which was there only if launching H2O backend from Python/R client. I think we want to have all combinations covered, but we can remove.

Reply:
h2o backend is jdk8+, jdk7 is supported only when MOJO+genmodel

Reply:
still raises the question if it makes sense to keep all clients combinations with jdk7: after all, the communication with backend relies only on  REST API, so I personally don't see the benefit of trying all Py versions with jdk7.
I added the new ones for consistency, but I'd be happy to keep only let's py2.7+jdk7 and py3.5+jdk7